### PR TITLE
added Material-UI

### PR DESCRIPTION
**Pull request by snaf-trip**

### CODE REVIEW

**Comment by mtalstykh-exadel** [comment]
please add some examples of usage the material-ui

File: package.json [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.12.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^1.1.5",
```
Comment (mtalstykh-exadel) [inline package.json]: That seems strange...

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,39 @@
+import React from 'react'
 import logo from './logo.svg';
 import './App.css';
+// импортируем элементы Material-UI
```
Comment (mtalstykh-exadel) [inline src/App.js]: replace with english

File: src/App.js [hunk 3]
```diff
@@ -1,23 +1,39 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      {/* Вставляем импортированный элемент */}
+      <Button variant="contained" color="primary" disableElevation className={classes.testButtons}>
+        Пример кнопки Material-UI
```
Comment (mtalstykh-exadel) [inline src/App.js]: and here too

**Review by tensenrey — DISMISSED** [review]

**Review by mtalstykh-exadel — DISMISSED** [review]

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Pull request merged**